src/components/FloatingButtons/FloatingButtons.js

```diff
@@ -24,15 +24,15 @@ import styled from 'styled-components/native';
 import SafeAreaView from 'react-native-safe-area-view';
 
 // Components
-import Icon from 'components/Icon';
+import Icon from 'components/modern/Icon';
 import Text from 'components/modern/Text';
 
 // Utils
 import { spacing } from 'utils/variables';
 
 // Types
 import type { ImageSource } from 'utils/types/react-native';
-import type { IconName } from 'components/Icon';
+import type { IconName } from 'components/modern/Icon';
```

Comment:
I assume `modern` means newer Icon component? If so – what do you think if we replace existing `Icon` into something like `LegacyIcon` and new one can stay `Icon` and be reused with new implementations?

Reply:
That's a good idea, however I would pause it until the code is actually merged with `develop`, otherwise keeping `vNext` branch in sync would be difficult.

But yeah, the general idea is that existing components keep using the LegacyIcon, while any new components should use modern/Icon.